frontend/packages/topology/integration-tests/features/export-application/export-of-applicaton.feature

```diff
@@ -1,83 +1,69 @@
-        @smoke @broken-test
+        @regression @odc-6738
         Scenario: Export application button in topology: EA-02-TC01
-            Given user has created a deployment workload "nodejs-ex-git-1"
-             When user navigates to Topology page
-              And user clicks on Export Application button
+            Given user has created "nodejs-ex-git-1" workload in "nodejs-ex-git-app" application
+             When user clicks on Export Application button
+              And user clicks on Ok button on Export Application modal to start the export
              Then user can see a toast message saying "Export of resources in aut-export-application has started."
-              And user can see a toast message saying "All the resources are exported successfully from aut-export-application. Click below to download it." and close it
+              And user can see a toast message saying "All the resources are exported successfully from aut-export-application. Click below to download it." with download option and close button
               And user can see primer job created in topology
 
 
         @regression @odc-6296
         Scenario: Export Application modal when export application is in progress: EA-02-TC02
-            Given user has created a deployment workload "nodejs-ex-git-2"
-              And user is at Topology page
+            Given user has created "nodejs-ex-git-2" workload in "nodejs-ex-git-app" application
              When user clicks on Export Application button
-              And user clicks on Export Application button again
+              And user clicks on Ok button on Export Application modal to start the export
+              And user clicks on Export Application button
              Then user can see "View logs" link, "Cancel Export", "Restart Export", and "Ok" button
-              # And user can see a toast message saying "All the resources are exported successfully from aut-export-application. Click below to download it." and close it
-#commented above line due to intermittent issue of not getting toast notification after export is complete
-
 
         @regression @manual @odc-6296
         Scenario: View logs when export application is in progress: EA-02-TC03
             Given user is at Topology page
-              And Export Application has already started
-             When user clicks on Export Application button again
+             When user clicks on Export Application button
               And user clicks on View logs link
              Then user can see page showing the Pod log tab
 
 
-        @regression @broken-test
+        @regression @odc-6738
         Scenario: Restart Export when export application is in progress: EA-02-TC04
             Given user is at Topology page
-              And Export Application has already started
-             When user clicks on Export Application button again
+             When user clicks on Export Application button
```

Comment:
```suggestion
             When user clicks on Export Application button
              And user clicks on Ok button on Export Application modal to start the export
```

Reply:
This scenario will run immediately after verifying the Restarts, Cancel, View Logs & Ok buttons

Reply:
But I'm getting flake where the export is already complete before clicking on the Export button to check for the restart button, and we need to make tests independent of each other to avoid any flakiness